GetDKAN/dkan
# Provide number widget in form

Re-roll/add test coverage for #4069 by @stefan-korn 

## QA steps

Add a field to the dataset.json schema file:

```json
    "number": {
      "title": "Number",
      "description": "A unique number for the dataset or API as maintained within an Agency catalog or database.",
      "type": "string",
      "minLength": 1
    }
```

now add settings to dataset.ui.json:

```json
  "number": {
    "ui:options": {
      "widget": "number",
      "step": 10
    }
  }
```

Go to create dataset form, confirm there is a number widget that increments by 10 on every click.
